src/foam/nanos/jetty/HttpServer.js

```diff
@@ -209,7 +210,10 @@ foam.CLASS({
         this.configHttps(server);
                 
         server.start();
-      } catch(Exception e) {}
+      } catch(Exception e) {
+        Logger logger = (Logger) getX().get("logger");
+        logger.error(e);
```

Comment:
http service starts really early, so test for null logger here and the one below please. thanks.

Reply:
fixed

Reply:
fixed